pkg/controller/user/user_controller.go

```diff
@@ -240,7 +268,7 @@ func (c *Controller) Start(stopCh <-chan struct{}) error {
 	return c.Run(4, stopCh)
 }
 
-func (c *Controller) encryptPassword(user *iamv1alpha2.User) (*iamv1alpha2.User, error) {
+func (c *Controller) ensurePasswordIsEncrypted(user *iamv1alpha2.User) (*iamv1alpha2.User, error) {
```

Comment:
Why give up mutatingadmissionwebhook to encrypt password?

Reply:
It is more appropriate to do this in the controller.